lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -699,7 +711,21 @@ static void on_ziti_event(ziti_context ztx, const ziti_event_t *event) {
             ev.identifier = instance->identifier;
             ev.operation = mfa_status_name(mfa_status_auth_challenge);
             CMD_CTX.on_event((const base_event *) &ev);
+            break;
+        }
+
+        case ZitiAPIEvent: {
+            api_update_req *req = calloc(1, sizeof(api_update_req));
+            req->wr.data = req;
+            req->ztx = ztx;
+            req->new_url = strdup(event->event.api.new_ctrl_address);
```

Comment:
should we be prepared for ziti API events that don't have a new controller address?

Reply:
future-proofed